src/documents/articles/a-quick-polymer-introduction.html.md

```diff
@@ -106,37 +146,37 @@ PRPL (pronounced purple) is a set of concepts to enable the next generation of w
 * Pre-cache - store the elements for remaining views
 * Lazy-load - when a new view is called, load the elements
 
-This pattern relies on [HTTP 2](https://http2.github.io) for the push capability (when you request only one element or page, the server recommends other files to cache to prevent having the browser work this out at a later time). The pre-cache relies on [service workers](http://www.w3.org/TR/service-workers/) to pull down (but not instantiate) all the other elements required for the routes. These elements will then be instantiated at the point in time they are needed (and all the code for this is in the browser).
+This pattern relies on [HTTP 2](https://http2.github.io) for the push capability (when you request only one element or page, the server recommends other files to cache to prevent having the browser work this out at a later time). The pre-cache relies on [service workers](http://www.w3.org/TR/service-workers/) to pull down (but not upgraded) all the other elements required for the routes. These elements will then be upgraded at the point in time they are needed (and all the code for this is in the browser).
 
-# Speeding up with vulcanisation
+## Speeding up with vulcanization
 
-As most browsers doesn't natively support HTML imports out-of-the-box (and HTTP 2 is not widely run on severs) they need to be polyfilled. The problem is polyfills are slower than native implementations, and worse you can import an element, that imports dependent elements, and that itself imports elements.
+As most browsers doesn't natively support HTML imports out-of-the-box they need to be polyfilled. The problem is polyfills are slower than native implementations, and worse you can import an element, that imports dependent elements, and that itself imports elements.
```

Comment:
- `As most browsers doesn't natively` to `Most browsers don't natively`
- `out-of-the-box they need to be polyfilled` to  `out-of-the-box so they will need to be polyfilled`
- `The problem is polyfills are` to `The problem with polyfills are that they are`
- `, and worse you can import an element, that imports dependent elements, and that itself imports elements.` to `. Worse than that, HTML Imports allows you to import an element, and then, in turn, that imported element now can import more dependent elements, and so on and so forth.`